work.py

```diff
@@ -123,11 +124,11 @@ def get_value(field, value):
         to_addr = []
         employees = [e.party.id for e in Employee.search([])]
 
-        uid = self.write_uid or self.create_uid
-        if uid:
-            discard_employees = [ce.party.id for ce in uid.employees
-                if not uid.send_own_changes]
-            employees = set(employees) - set(discard_employees)
+        work_uid = self.write_uid or self.create_uid
+        user_id = Transaction().user
+        user = User(user_id)
+        if work_uid == user and not user.send_own_changes and user.employee:
+            employees = set(employees) - set([user.employee.party.id])
```

Comment:
I do not understand why this change should fix the problem. Or why it currently does not work.